lib/sqlalchemy/engine/result.py

```diff
@@ -1582,7 +1607,9 @@ def freeze(self) -> FrozenResult[_TP]:
 
         return FrozenResult(self)
 
-    def merge(self, *others: Result[Any]) -> MergedResult[_TP]:
+    def merge(
+        self, *others: Result[Unpack[_RawRowType]]
```

Comment:
**Michael Bayer** (zzzeek) wrote:

_RawRowType should be TupleAny for both here

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986